cucumber/dev-base/linux-headers/linux-headers.buildinfo

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2016 Scott Court
```

Comment:
Please make sure to add your name and the current year here, i.e.
`# Copyright 2018 John Doe`